src/main/resources/templates/pps/home.html

```diff
@@ -71,7 +70,6 @@ <h1 class="govuk-heading-xl" id="main-heading">
                             <p class="govuk-body">This payment method usually takes longer to process (up to 4 working
                                 days). <a href="late-filing-penalty/bank-transfer/which-penalty-service" class="govuk-link">Find out which
                                     account details to use for a bank transfer</a>.
-                                <!--Waiting for SAN-71 to complete link above (line 72)--></p>
```

Comment:
Can you check the paragraph end tag needs adding back - looks to be accidentally deleted?